### DISCUSSION TITLE
就是说内存优化也太逆天了吧

### DISCUSSION THREAD

**Discussion by ZS2147483647** [Meloong-Git/PCL#4459] [讨论]
我7.1GiB内存占用,优化完了变成4GiB,是不是有点离谱呢,这功能要不出个单独插件放启动文件夹里

**Comment by ZS2147483647** [comment]
或者出个命令让游戏启动前优化一遍

**Reply by zkitefly** [reply]
内存调整页面有启动前内存优化开关

**Comment by zkitefly** [comment]
> 我7.1GiB内存占用,优化完了变成4GiB,是不是有点离谱呢,这功能要不出个单独插件放启动文件夹里

不是有启动器参数 `--memory` 吗

**Comment by sulingjiang** [comment]
>我7.1GiB内存占用,优化完了变成4GiB,是不是有点离谱呢,这功能要不出个单独插件放启动文件夹里

猜你需要：mem reduct
